CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.17.0 (February 17, 2025)
+
+NOTES:
+
+* **This release requires Juju controller version 2.9.49 or higher juju.**
+* **If using JAAS, this release requires Juju controller version 3.5.0 or higher.**
+* This release uses juju client api code from the juju 3.6.0 release.
+
+ENHANCEMENTS:
+
+* Improvements to documentaton (Thanks @thihoc).
+* Schema generation on install command (Thanks @SimoneDutto).
+
+BUG FIXES:
+
+* fix: deploy from a closed channel #661 which addresses issue #647
+* fix: get plan from req.Plan instead of req.State #665 which addresses issue #664
```

Comment:
Some bug descriptions like this one could be cleaned up to describe the bug that was fixed rather than the implementation detail.
Also the numbers aren't actual links which is not a big deal but maybe we can turn them into links in the future.